packages/core/test/xhrProxy.spec.ts

```diff
@@ -119,31 +127,13 @@ describe('xhr proxy', () => {
     })
   })
 
-  it('should track successful sync request', (done) => {
```

Comment:
This test isn't needed anymore?

Reply:
I don't see how we could replicate this behavior with a stub.
However, FMU it does not change our proxy behavior, so it seemed fine to me to drop it.